source/administration/config-settings.rst

```diff
@@ -1824,6 +1824,14 @@ Secure Amazon S3 Connections
 | This feature's ``config.json`` setting is ``"AmazonS3SSL": true`` with options ``true`` and ``false`` for above settings respectively.                               |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable File Attachments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When false, disables file and image uploads on messages.
```

Comment:
Maybe this is better: "When false, users cannot upload and share files and images by attaching them to messages."